crates/cast/src/starknet_commands/account/add.rs

```diff
@@ -0,0 +1,101 @@
+use crate::starknet_commands::account::{prepare_account_json, write_account_to_accounts_file};
+use anyhow::{ensure, Result};
+use camino::Utf8PathBuf;
+use cast::get_chain_id;
+use cast::helpers::response_structs::AccountAddResponse;
+use cast::helpers::scarb_utils::CastConfig;
+use clap::Args;
+use starknet::core::types::BlockTag::Latest;
+use starknet::core::types::{BlockId, FieldElement};
+use starknet::providers::{
+    jsonrpc::{HttpTransport, JsonRpcClient},
+    Provider,
+};
+use starknet::signers::SigningKey;
+
+#[derive(Args, Debug)]
+#[command(about = "Add an account to the accounts file")]
+pub struct Add {
+    /// Name of the account to be added
+    #[clap(short, long)]
+    pub name: String,
+
+    /// Address of the account
+    #[clap(short, long)]
+    pub address: FieldElement,
+
+    /// Class hash of the account
+    #[clap(short, long)]
+    pub class_hash: Option<FieldElement>,
+
+    /// Account deployment status
+    /// If not passed, sncast will check whether the account is deployed or not
+    #[clap(short, long)]
+    pub deployed: bool,
+
+    /// Account private key
+    #[clap(long)]
+    pub private_key: FieldElement,
```

Comment:
I wonder if we should allow for getting this from file as well. Potentially it can be a security issue to have private keys in plain text in terminal, so some people may want to be able to provide it in some more secure way

Reply:
Maybe in the future? 😅

Reply:
hmm, sure, but let's create an issue for this so it won't get lost